core/src/main/scala/org/broadinstitute/dsde/rawls/expressions/SlickExpressionEvaluator.scala

```diff
@@ -210,7 +210,7 @@ private[expressions] class SlickExpressionEvaluator protected (val dataAccess: D
   // Determine which FinalFunc to use by checking if entity attribute is a reserved attribute
   private def entityAttributeFinalFunc(attrNameContext: AttributeNameContext): FinalFunc = {
     val attrName = toAttributeName(attrNameContext)
-    if (Attributable.reservedAttributeNames.contains(attrName.name)) {
+    if (Attributable.reservedAttributeNames.exists(_.equalsIgnoreCase(attrName))) {
```

Comment:
This is the source of the test failure. The question is, if someone queries "library:name", should this give the same result as querying "default:name"? (Existing behavior is that these 2 expressions are the same). My vote is that a query of library:name should not qualify as a reserved attribute in an expression, it would create problems if you were trying to query the tdr:name field in your table and Rawls was interpreting the query as a "name" query